includes/class-wc-payments-account.php

```diff
@@ -340,18 +340,6 @@ public function maybe_handle_oauth() {
 			return;
 		}
 
-		if ( isset( $_GET['wcpay-connection-success'] ) ) {
-			$account_status = $this->get_account_status_data();
-
-			if ( empty( $account_status['error'] ) && $account_status['paymentsEnabled'] ) {
-				$this->redirect_to_wc_admin_home();
-			} else {
-				$message = __( 'Thanks for verifying your business details!', 'woocommerce-payments' );
-				$this->add_notice_to_settings_page( $message, 'notice-success' );
-			}
-			return;
-		}
-
```

Comment:
IMO, we don't need this logic anymore since we are passing the return URL while connecting to stripe. Let me know if I am missing anything edge case.

Reply:
One change that I see is that confirmation message after the update action is not shown anymore. Could be not critical for MVP though. @LevinMedia what do you think?